specs/resources/accounts/createAccountFundingSource.yml

```diff
@@ -29,7 +29,9 @@ post:
   responses:
     '201':
       description: successful operation
-      headers: {}
+      headers:
+        Location:
+          $ref: '../../schemas/commonSchemas.yml#/components/headers/Location'
       content:
```

Comment:
@ShreyaThapa I'm curious if we can remove `content` from 201 responses? Doesn't content determine how the response should be deserialized?

Reply:
nvm, it looks like we're returning that media type in the response, even though it's an empty body

Reply:
Ah yes! Good point! I was simply adding the Location header and overlooked the content-type. I will remove them for 201 responses in a new commit!

Reply:
I see what you mean in the first comment. I guess we're returning it for all responses.

Reply:
We decided to remove content from responses that have empty bodies. Addressed in this [commit](https://github.com/Dwolla/dwolla-openapi/pull/12/commits/8c4979338155fe2c06af52b29ac6e5c1976bfb45).